### DISCUSSION TITLE
Flagged

### DISCUSSION THREAD

**Discussion by mabh5** [community/community#34498] [Other Feature Feedback, Questions, & Ideas]
my account has been flagged. please resolve

**Comment by nalsur-veallam** [comment]


**Reply by mabh5** [reply]
The situation has been resolved. Best guess is that one of their automated systems thought my account was a fake and flagged it

**Comment by jfullstackdev** [comment]
try Support 

https://support.github.com/contact/account

do you have any idea why you account was flagged ?

**Reply by mabh5** [reply]
The situation has been resolved. Best guess is that one of their automated systems thought my account was a fake and flagged it
